pytest_tank_test/__init__.py

```diff
@@ -71,7 +77,9 @@ def pytest_configure(config):
 
     # Add the <current-repo>/tests/python folder to the PYTHONPATH so custom
     # python modules from it can be used in the tests.
-    if os.path.basename(repo.name).lower() != "tk-core":
+    # If we're running tests inside tk-core, we shouldn't add it as tk-toolchain
```

Comment:
This also fixes an issue where azure pipeline does not use the repo name as the folder a repo is cloned into.